omod/src/main/webapp/dictionary/concept.jsp

```diff
@@ -227,7 +227,7 @@
                 <c:forEach items="${command.locales}" var="loc">
                     <td class="${loc}">
                         <c:forEach items="${command.conceptAnswersByLocale[loc]}" var="answer">
-                            <a href="concept.htm?conceptId=${fn:substring(answer.key, 0, fn:indexOf(answer.key, '^'))}"><c:out value="${answer.value}" /> (${fn:substring(answer.key, 0, fn:indexOf(answer.key, '^'))})</a><br/>
+                            <a href="concept.htm?conceptId=${fn:substring(answer.key, 0, fn:indexOf(answer.key, '^'))}">${answer.value}(${fn:substring(answer.key, 0, fn:indexOf(answer.key, '^'))})</a><br/>
```

Comment:
@wluyima Did you mean adding a retired class like in here, https://github.com/openmrs/openmrs-module-legacyui/blob/master/omod/src/main/java/org/openmrs/web/controller/ConceptFormController.java#L1046 
Even if the retired class is added, along with <c:out the HTML code is being displayed.